test/v1/models/response/retrieveFinancialDetails/PaySpec.scala

```diff
@@ -26,7 +26,7 @@ class PaySpec extends UnitSpec {
       |{
       |  "taxablePayToDate": 100.11,
       |  "totalTaxToDate": 102.11,
-      |  "payFrequency": "CALENDAR MONTHLY",
+      |  "payFrequency": "monthly",
```

Comment:
Isn't the downstream value still "CALENDAR MONTHLY"? You might need to introduce enums PayFrequency and DownstreamPayFrequency to do the conversion.